apps/studio/src/server/modules/page/__tests__/page.router.test.ts

```diff
@@ -2015,4 +2016,72 @@ describe("page.router", async () => {
       })
     })
   })
+  describe("scheduled publishing", () => {
```

Comment:
can rename it from `scheduled publishing` to the endpoint name itself `schedulePage` for consistency